docker-compose.override.yaml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+
+services:
+    composer:
+        volumes:
+            - ./composer.json:/app/composer.json
+            - ./composer.lock:/app/composer.lock
+            - ./symfony.lock:/app/symfony.lock
```

Comment:
had some success in using `pipenv` through the related image and mounted volumes, to use a consistent version rather than relying on the host's one. Could be done on this if it mounted the folders Symfony Flex writes to?